platform-sdk/swirlds-common/src/main/java/com/swirlds/common/crypto/Hash.java

```diff
@@ -48,19 +48,33 @@ public Hash() {
     }
 
     /**
-     * Same as {@link #Hash(byte[], DigestType)} but with an empty byte array.
+     * Same as {@link #Hash(Bytes, DigestType)} but with an empty byte array.
      */
     public Hash(@NonNull final DigestType digestType) {
-        this(new byte[digestType.digestLength()], digestType);
+        this(Bytes.wrap(new byte[digestType.digestLength()]), digestType);
```

Comment:
Wrapping in Bytes will affect hashing which instantiates Hashes orders of magnitude more than the rest of the system.
The performance impact needs to be evaluated.

Reply:
please see my other comment